quickstart.sh

```diff
@@ -230,4 +242,4 @@ echo "  The TestRail background task is also automatically started 60s after the
 echo
 echo
 echo "To shut down TestRail again run the following command in this folder:"
-echo "docker-compose down -v"
\ No newline at end of file
+echo "docker-compose/docker compose down -v"
```

Comment:
@acuanico-tr-galt These doesn't seem related to chrome-php updates. Is this for your local only?

Reply:
@parboleda-tr-galt I included these updates to support newer version of docker compose (Compose v2).

Reply:
Let's separate this docker-compose changes to a separate PR